lib/movable_ink/aws.rb

```diff
@@ -39,7 +39,10 @@ def run_with_backoff
                Aws::AutoScaling::Errors::ThrottledException,
                Aws::S3::Errors::SlowDown,
                Aws::Route53::Errors::ThrottlingException,
-               Aws::SSM::Errors::TooManyUpdates
+               Aws::Route53::Errors::ServiceError,
+               Aws::SSM::Errors::TooManyUpdates,
+               Aws::Route53::Errors::InvalidChangeBatch,
+               Aws::Route53::Errors::InvalidInput
```

Comment:
These two are not throttling responses.  It seems that they should be handled, but not necessarily by retrying the request.  Presumably if the request was invalid, trying it again will not make it valid.